processing/src/main/java/io/druid/query/aggregation/BufferAggregator.java

```diff
@@ -178,4 +178,26 @@ default void relocate(int oldPosition, int newPosition, ByteBuffer oldBuffer, By
   {
   }
 
+  /**
+   * Returns true if the aggregator is nullable and the aggregated value is null
+   * <p>
+   * <b>Implementations must not change the position, limit or mark of the given buffer</b>
+   * <p>
+   * Implementations are only required to support this method if they the aggregator supports null values.
+   * If it doesn't support null always returning false is recommended.
```

Comment:
This phrase doesn't make a lot of sense to me, if aggregator doesn't support null values returning true from this method would be a straightaway mistake, though it's obvious and IMO shouldn't be mentioned